### PR TITLE
Change readme

### PR DESCRIPTION
**Pull request by Smolevich**
- Replace old links on actual
- Little formatting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1073,4 +1074,4 @@ By default, Laravel keeps a log in memory of all queries that have been run for
 DB::connection()->disableQueryLog();
 ```
 
-*From: http://laravel.com/docs/database#query-logging*
+*From: https://laravel.com/docs/4.2/database#query-logging*
```
[the remark this answers is not in the document]
Comment (Smolevich) [reply README.md]: @jenssegers , do you want change this link ?

[the remark this answers is not in the document]
Comment (Smolevich) [reply README.md]: @jenssegers ?

[the remark this answers is not in the document]
Comment (Smolevich) [reply README.md]: @jenssegers, can you answer me ?

**Comment by Smolevich** [comment]
@jenssegers, check please

**Comment by Smolevich** [comment]
@jenssegers, any chance that you review it?

**Comment by jenssegers** [comment]
Some documentation links are still linking to old versions of the documentation. Are there updated versions of those?

**Comment by Smolevich** [comment]
If info by old link exists in new version docs, then yes, if no, i kept old link

**Pull request merged**